api/src/main/java/org/pmiops/workbench/exceptions/ExceptionAdvice.java

```diff
@@ -28,6 +28,10 @@
             ErrorResponse errorResponse = ((BadRequestException) e).getErrorResponse();
             return ResponseEntity.status(statusCode).body(
                 ((BadRequestException) e).getErrorResponse());
+          } else if (e instanceof ForbiddenException) {
+            ErrorResponse errorResponse = ((ForbiddenException) e).getErrorResponse();
```

Comment:
opt: common base class or interface for these exceptions would likely be cleaner. Probably all of our exceptions setting an `ErrorResponse` (even without a specific code) would make sense, i.e. common base class.

Reply:
Yeah, I can make a base class and just handle it all the same way.